lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -361,3 +361,78 @@
     puts 'No gettext-setup gem found, skipping GettextSetup config initialization' if Rake.verbose == true
   end
 end
+
+def create_gch_task(changelog_user, changelog_project, changelog_since_tag, changelog_tag_pattern = 'v%s')
```

Comment:
Could you add `= nil` for first 3 arguments? The code handles it and it'd benefit users who don't want to override anyway.

Reply:
:+1: